blocks/hero/hero.js

```diff
@@ -1,7 +1,28 @@
 import { decorateIcons } from '../../scripts/lib-franklin.js';
 import { addClipboardInteraction } from '../../scripts/utils.js';
 
+function addBackLink(block) {
+  if (window.location.pathname.endsWith('/news-und-events/')) {
+    return;
+  }
+
+  const bl = document.createElement('div');
+  // The 2 elements are created in the 'backward' order to allow the hover over the link
+  // to change the color of the arrow. They are put in the correct position via the
+  // 'order' attribute in the CSS.
+  bl.innerHTML = `<a href="/de/semiconductor-manufacturing-technology/news-und-events/" class="back-link"> Zurück zu Informationen und Veranstaltungen</a>
```

Comment:
You could explore using [Franklin placeholders](https://www.hlx.live/docs/placeholders) here to avoid hardcoding strings. For the `href` though we'd need to distinguish and here we can use the `getMetadata('locale') to get the locale.

Reply:
I didn't do this yet as it would create merge conflicts with #89 I think its best to do this once that PR is merged. Hope that's ok.

Reply:
> I didn't do this yet as it would create merge conflicts with https://github.com/hlxsites/zeiss/pull/89 I think its best to do this once that PR is merged. Hope that's ok.

Makes sense. #89 is ready to be merged but needs an approval.